os/arch/arm/src/amebad/amebad_serial.c

```diff
@@ -114,7 +114,7 @@
 #elif defined(CONFIG_UART1_SERIAL_CONSOLE)
 #define CONSOLE_DEV             g_uart1port             /* UART1 is console */
 #define TTYS0_DEV               g_uart1port             /* UART1 is ttyS0 */
-#define CONSOLE                 UART1_DEV
+#define CONSOLE                 UART3_DEV
```

Comment:
In the comments of amebad_serial.c, UART3_DEV is KM0 luart.
```
/*
 * Available UARTs in AmebaD
 * UART0_DEV: KM4 uart0
 * UART1_DEV: KM4 uart1_bt
 * UART2_DEV: KM0 log uart
 * UART3_DEV: KM0 luart
 */
```
UART1 is connected to KM0? not KM4?

Reply:
Yes, the UART1 actually associates with the KM0 luart.
If necessary, I will submit another PR to completely change all the UART1 notation to UART3 instead.